src/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -493,7 +493,7 @@ export class MongoEntityConverter {
     }
   }
 
-  public convertToDevice(mongoDevice: MongoDevice): Device {
+  public convertToCoreDevice(mongoDevice: MongoDevice): CoreDevice {
```

Comment:
These are still just "Devices", not "CoreDevices". 
In the eye of Sofie, "CoreDevices" doesn't exist. It's all simply Devices.